### DISCUSSION TITLE
Understanding Kdown

### DISCUSSION THREAD

**Discussion by elidamd** [UMEP-dev/UMEP#556] [General]
Hi,
I am using TestReferenzJahr for climate data input in my simulations.
This is a dataset provided by DWD and relates to an average of 20 years of measurements, resulting in a dataset for a typical year.
This dataset includes only direct and diffuse radiation (horizontal plane).

Based on this: "direct-beam radiation = (global−diffuse) / sin(sun altitude)", 
I am using NOAA calculator to find altitude and convert the direct radiation given into kdir. 
In sequence, I assume kdown as kdir + kdif. 

I am simulating for wintertime and comparing results (kdown I calculated x kdown calculated by Solweig). They seem much different, so I would like to understand how kdown was calculated in Solweig please.
Also, which is the correct value expected for kdown (given direct and diffuse radiation):
- kdir (beam) + kdiff 
- OR direct rad. (horizontal) + kdiff.

Thanks!
[kdown winter.xlsx](https://github.com/UMEP-dev/UMEP/files/12763154/kdown.winter.xlsx)

Best,
Élida

**Comment by biglimp** [comment]
Solweig make use of direct-beam as input. Kdir = direct-beam from inputdata. We have that included since we have an option to calculate direct-beam and diffues from only global radiation. Kdown is the actual radiation at the surface included reclection + sky duffuce + direct for each pixel and in your case a assume from your POI.

It looks like maybe your POI is in shadow. Could that be the case?

**Reply by elidamd** [reply]
Thanks for the reply! 
In kdown (necessary for input data), the direct component of kdown is not the direct-beam, but related to horizontal surface? 
If I don't have a value for reflected radiation, I will consider just the diffuse and direct to horizontal surface. Is that correct?
I don't think that the POI is under shadow, it is a point in open park, in a simulation without trees. The neighbor building has 1 storey and is 60m far.

**Reply by biglimp** [reply]
kdown is global radiation. See the table here: https://umep-docs.readthedocs.io/en/latest/pre-processor/Meteorological%20Data%20MetPreprocessor.html

The reflected radiation is caclulated in the model so you dont have to consider that.

If you save out the shadow maps you can examine if your point is in shadow at any point during the simulation.

**Reply by elidamd** [reply]
I don't have a value for the global radiation in my dataset, only direct and diffuse. 
Given this, as input for global radiation, should I sum both values that are given on a horizontal surface (direct and diffuse), or should I sum diffuse + direct-beam radiation?

**Reply by biglimp** [reply]
Yes, Kdown=direct (horizontal) + diffuse

**Reply by elidamd** [reply]
Super, many thanks!!
